lib/pymedphys/mudensity.py

```diff
@@ -1,30 +1,23 @@
-"""A suite of functions to determine the MU Density for a range of
-formats
+from ._vendor.deprecated import deprecated as _deprecated
```

Comment:
this answers my question earlier about import of this and deprecation.
Is it feasible to indicate at what version this will be obsolete as a vigorous warning to the user/programmer to change their code?

Reply:
See https://github.com/pymedphys/pymedphys/pull/1168#discussion_r520927280, let me know what you think.